linkernetworks/azure-intelligent-edge-patterns
# Backend/peter dev

## Purpose

* Now setting page should be able to see something...

## Does this introduce a breaking change?

```
[] Yes
[v] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[v] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code
*  Run [Readme instructions](factory-ai-vision/EdgeSolution/modules/WebModule/backend/cameras)
*  Go to Settings and see the configs are shown

## UnitTest the code

```
python manage.py test cameras/tests
```

## What to Check
Verify that the following are valid
* ...

## Other Information
